### DISCUSSION TITLE
If Flet had a page designer, it could be on par with QT！

### DISCUSSION THREAD

**Discussion by 188224912** [flet-dev/flet#2598] [Ideas]
If Flet had a page designer, it could be on par with QT。This is very friendly to a novice like me，Highly anticipated！

**Comment by FeodorFitsner** [comment]
Thanks for your support!

We are absolutely going to make visual designer for Flet!

**Reply by 188224912** [reply]
I am extremely looking forward to

**Comment by asmaier** [comment]
Have a look at https://labdeck.com/python-designer/flet-gui-designer/ .

**Reply by ap4499** [reply]
Aside from the incomprehensible product/pricing page.. this looks pretty cool.

Working out what version you need is an entire side quest in itself!

**Reply by arthur-tacca** [reply]
Looks like it sticks everything in a Stack and gives them all explicit x,y positions (judging by the generated code shown there ... although that's for Tk not Flet!). Sometimes you need to do this but the vast majority of Flet code should be laid out with columns and rows.

**Comment by yunguangli** [comment]
With AI , it is a lot easier now. I just sketch UI on paper, take a photo, and instruct AI to generate UI in Flet. The Flet version in AI is a bit old but it is easy to fix.

**Reply by arthur-tacca** [reply]
This works OK but usually does things with some extra clutter (e.g. unnecessarily nesting multiple layers of Container objects). I sometimes use this approach but write the code I actually use by hand, using the generated code for inspiration.

**Comment by baruchRepo** [comment]
There are already several low-code projects for flet, but an official one would be great! Like Flutterflow for Flutter.

**Comment by ChinaGodzilla** [comment]
absolutely ,this is good idea.

**Comment by ndonkoHenri** [comment]
A [community member](https://discord.com/channels/981374556059086931/1435827873654112419/1435827873654112419) made [this](https://fletboxweb.onrender.com/home) recently.
